environment.yaml

```diff
@@ -0,0 +1,32 @@
+name: default-datahub-user-image
+# This is the standard conda configuration file. Use this file to list
+# the conda packages that you need installed in your environment.
+channels:
+  - conda-forge
+
+dependencies:
+
+# Items required for basic level functionality
+- python==3.11.*
+- git==2.46.0
+- jupyter-resource-usage=1.1.0
+- jupyterhub==4.1.6
+- jupyterlab==4.2.5
+- jupyter_server==2.14.2
+- notebook==7.2.2
+- nbgitpuller==1.2.1
+
+# vscode
+- code-server==4.23.1
+- jupyter-vscode-proxy==0.6
+
+# other packages
+- seaborn==0.13.2
+- altair==5.4.1
+- ibis-framework[pandas,duckdb]==9.3.0
+- leafmap==0.36.10
+- jupyterlab-myst==2.4.2
```

Comment:
Don't need the stuff from lines 19-28 in a default image, also can you think of anything we might want to throw in there?

Reply:
@balajialg any thoughts?  this is the template for NEW hub images, and not related to us porting our old images to the new paradigm

Reply:
here's the rough (read: VERY rough) docs for how to contribute to this repo:  https://github.com/berkeley-dsep-infra/stat20-user-image/blob/main/CONTRIBUTING.md